test/blackbox_tests/outbound_redirect_test.go

```diff
@@ -0,0 +1,85 @@
+package blackbox_tests_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"runtime"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/kumahq/kuma-net/iptables/builder"
+	"github.com/kumahq/kuma-net/iptables/config"
+	"github.com/kumahq/kuma-net/test/framework/ip"
+	"github.com/kumahq/kuma-net/test/framework/netns"
+	"github.com/kumahq/kuma-net/test/framework/socket"
+	"github.com/kumahq/kuma-net/test/framework/tcp"
+)
+
+var _ = Describe("Inbound TCP traffic from all ports", func() {
```

Comment:
It's called 'Inbound' but the test is actually for Outbound.

Reply:
addressed